### ISSUE TITLE
Bug in LDP when retrieving a NonRDFSource

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/active_fedora#704]
```
instance variable @graph not defined
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldp-0.2.1/lib/ldp/response.rb:69:in `remove_instance_variable'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldp-0.2.1/lib/ldp/response.rb:69:in `block in dup'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldp-0.2.1/lib/ldp/response.rb:64:in `tap'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldp-0.2.1/lib/ldp/response.rb:64:in `dup'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/caching_connection.rb:77:in `cache_resource'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/caching_connection.rb:11:in `get'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldp-0.2.1/lib/ldp/resource.rb:43:in `get'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/file.rb:272:in `retrieve_content'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/file.rb:103:in `remote_content'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/datastreams/nokogiri_datastreams.rb:68:in `remote_content'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/datastreams/nokogiri_datastreams.rb:24:in `ng_xml' /Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/om-3.1.0/lib/om/xml/document.rb:119:in `find_by_xpath'", "/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/om-3.1.0/lib/om/xml/document.rb:139:in `find_by_terms'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/om_datastream.rb:116:in `find_by_terms'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/om-3.1.0/lib/om/xml/term_value_operators.rb:12:in `term_values'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/delegated_attribute.rb:64:in `read_om'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/delegated_attribute.rb:48:in `reader'
/Users/justin/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/bundler/gems/active_fedora-cc1ab1b4e572/lib/active_fedora/attributes.rb:95:in `array_reader'
```

**Comment by jcoyne** [comment]
See https://github.com/cbeer/ldp/commit/82ba9bb673b9e160af62ad2becab4f620fa0b071. Looks like we may need to check if the instance variable exists before we clear it.

**Comment by jcoyne** [comment]
fixed by https://github.com/cbeer/ldp/pull/43

**Comment by cam156** [comment]
@jcoyne I would merge you fix if I could.  Thanks for jumping on it!

**Comment by jcoyne** [comment]
@cam156 put :+1: on the LDP pr

**Comment by jcoyne** [comment]
Closed by LDP 0.2.2
